src/lib/_imports/components/c-message--expanded.css

```diff
@@ -33,19 +33,17 @@
 
 /* Structure */
 
-:--c-message--scope-section {
+/* Skin */
+@define-mixin c-message--scope-section {
+  color: var(--global-color-primary--dark);
+
+  /* Structure */
   --pad-vert: 20px;
   --pad-horz: 30px;
 
   border-width: var(--global-border-width--normal);
   border-style: solid;
 }
-
-/* Skin */
-
-@define-mixin c-message--scope-section {
-  color: var(--global-color-primary--dark);
-}
```

Comment:
Given the cognitive overhead of the refactor and novelty of introducing mixins, I'm fine dropping the [`/* Skin */` and `/* Structure */` comments/organization](https://tacc-main.atlassian.net/wiki/spaces/~wbomar/pages/7277544/CSS+Style+Guide+-+Architecture+-+Structure+Skin) for now.